src/main/java/frc/robot/subsystems/gpm/Intake.java

```diff
@@ -30,7 +32,10 @@ public class Intake extends SubsystemBase {
     public enum Mode {
         DISABLED(0,0),
         INTAKE(.3,.3),
-        REVERSE(-.3,-.3);
+        REVERSE(-.3,-.3),
+        PickedUpNote (.3, .3),
+        Pause(0,0),
+        Wait(.3, .3);
```

Comment:
We shouldn't have this many intake modes. INTAKE, REVERSE, and DISABLED are useful, but we don't need the rest.

Reply:
We need the rest. We do not want random state variables.